cmd/main.go

```diff
@@ -68,6 +83,15 @@ func init() {
 	rootCmd.Flags().Int64Var(&maxVolumesPerNode, "maxvolumespernode", 0, "limit of volumes per node")
 	rootCmd.Flags().StringVar(&shareRelistInterval, "share-relist-interval", "",
 		"the time between controller relist on the share resource expressed with golang time.Duration syntax(default=10m")
+	rootCmd.Flags().BoolVar(&readOnly, "readonly", false, "does not watch for resource updates")
```

Comment:
`readonly` does not make sense to me here with the `does not watch for resource updates` description. What is it setting as `readonly`? Maybe it should be something like `no-watch`?

Reply:
I agree, the name `read-only` doesn't resemble what the feature does. However, `no-watch` also doesn't either, given with this flag enabled it will only watch for `Share` resources, no longer for `Secret` and `ConfigMap`.

What other names would we suggest for this flag?

Reply:
perhaps to verbose, but `no-watch-since-read-only` ?

Reply:
Maybe the term "on demand" can be illustrative enough to portray the operator will populate the cache "on demand", just before mounting the actual volume.

Thus, I would suggest `--on-demand` for the flag name.

Reply:
hmm ... yeah I think `--on-demand` is too far off topic

I'm still leaning toward some sort of longer name that mentions not watching because we will not update the volume after initial creation

maybe `--volume-write-create-only`

or if our default is going to be this new mode, where we are read only and you have to opt into updating volumes after their initial creation, we go with 

`--update-volume-after-create`

@adambkaplan - any opinions ?

Reply:
How about `--static-volume`?

The inverse feels easier to make a flag for, like `--reload-resources` to imply that we can reload the backing resources.

Reply:
I like the idea of using `--reload-resources` and inverting the logic currently applied 👍🏼  What do you think, @gabemontero, @coreydaley?

Reply:
I could go with `--reload-resource`.  @adambkaplan is good at names :-) 

That said, using his pivot as a building block, what does everyone think about `--refresh-resources` ?  First blush I like that even better.  But if nobody else does, I'm fine with reload.

And of course inverting the flow, so that the default is *NOT* to refresh/reload, as we've discussed in various arch/planning/etc calls is the path we want to hop on.

Reply:
I'm good with where we are here now ... thanks everyone